src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -104,6 +104,14 @@
         (rr/status ((response :outcome) outcomes->codes))
         (rr/content-type "text/plain"))))
 
+(schema/defn handle-get-ca-expirations
+  [ca-settings :- ca/CaSettings]
+  (let [response {:ca-certs (ca/ca-expiration-dates (:cacert ca-settings))
+                  :crls (ca/crl-expiration-dates (:cacrl ca-settings))}]
```

Comment:
Would it be more or less confusing if the response keys more-closely matched the settings keys, e.g. `:cacerts` and `cacrls`?

Reply:
I didn't want to do `cacrls` because there isn't really any such thing as a " CA crl". We make the distinction in the settings because we keep the CRL two places, one for the CA and another for the client. But they have the same expiration, it's just two copies of the same file.

No strong opinion on whether to keep or get rid of the hyphen. In general, only the very oldest settings do not have hyphens. In newer code, we tend to use them (or underscores) to make things easier to read. Since this info isn't directly tied to Puppet's settings at all, I decided to add one.

Reply:
Fair. I don't have strong feelings.